### DISCUSSION TITLE
Operation Workflow Error when retrying or starting a workflow with the same name.

### DISCUSSION THREAD

**Discussion by kclaes** [argoproj/argo-workflows#9862] [Q&A]
We're seeing an increased occurrence of not being able to retry a failed workflow, not even when deleting the workflow from the system and recreating it with the same name (!). 
Only when we resubmit, given it a new name, does the workflow start successfully.

The workflow will fail almost instantaneously with a "workflow operation error" and the UI will show links to missing nodes, as shown below.
![image](https://user-images.githubusercontent.com/1501824/196662642-2896183a-778a-4f35-b63f-cc1ca940fc0c.png)

The node itself shows like this:
![image](https://user-images.githubusercontent.com/1501824/196662742-f5f8e281-fdda-4f60-905a-dd712bf7b2f1.png)

The problem is that our workflows are fairly large, contain confidential information and I haven't been able to reproduce the behavior in a smaller, more concise example. The controller pod shows no errors of use regarding this workflow.

Basically asking if anyone has seen something similar and for pointers on how to determine what could be going wrong.

If anybody has an idea about which saved/cached part of Argo "remembers" the faulty workflow state so that we can't even create one with the same name after deleting it with `kubectl delete`, would be highly appreciated!

k8s provider is EKS and argo is at 3.4.1.
